pwa/src/pages/configurationsExportButton/index.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react"
```

Comment:
Adding this page was exactly what we wanted. However, on thing should be updated.

You currently named the page itself `configurationsExportButton`, but that's not a correct name for a page, because that's just the component it is (currently) including. 

The correct naming should be (and the folder names should reflect that): `configurations`

So the path of this file would become `/src/pages/configurations/index.tsx`